cleaners/opera.xml

```diff
@@ -119,19 +127,19 @@ Notes about Opera
     <action command="delete" search="walk.files" path="$$profile$$\JumpListIconsOld\"/>
     <action command="delete" search="walk.files" path="$$profile$$\JumpListIcons\"/>
     <!-- below are entries valid before Opera 15 -->
-    <action command="delete" search="glob" path="$appdata\Opera\Opera*\download.dat"/>
+    <action command="delete" search="file" path="$appdata\Opera\Opera*\download.dat"/>
     <action command="delete" search="file" path="$appdata\Opera\Opera\profile\download.dat"/>
     <action command="delete" search="file" path="~/.opera/download.dat"/>
-    <action command="delete" search="glob" path="$appdata\Opera\Opera*\search_field_history.dat"/>
-    <action command="delete" search="glob" path="~/.opera/search_field_history.dat"/>
+    <action command="delete" search="file" path="$appdata\Opera\Opera*\search_field_history.dat"/>
+    <action command="delete" search="file" path="~/.opera/search_field_history.dat"/>
     <!-- global.dat is in Opera 9, global_history.dat is in Opera 10 -->
     <action command="delete" search="file" path="$appdata\Opera\Opera\profile\global.dat"/>
     <action command="delete" search="file" path="$appdata\Opera\Opera\profile\typed_history.xml"/>
     <action command="delete" search="file" path="$appdata\Opera\Opera\profile\vlink4.dat"/>
     <action command="delete" search="file" path="$localappdata\Opera\Opera\profile\vps\????\md.dat"/>
-    <action command="delete" search="glob" path="$appdata\Opera\Opera*\global_history.dat"/>
+    <action command="delete" search="file" path="$appdata\Opera\Opera*\global_history.dat"/>
     <action command="delete" search="glob" path="$appdata\Opera\Opera*\typed_history.xml"/>
-    <action command="delete" search="glob" path="$appdata\Opera\Opera*\vlink4.dat"/>
+    <action command="delete" search="file" path="$appdata\Opera\Opera*\vlink4.dat"/>
```

Comment:
Same comments apply to these changes as the above about `file` vs `glob`

Reply:
Yes!